packages/cli/src/lib/cli/cliProcess.js

```diff
@@ -36,12 +36,12 @@ module.exports = class CLIProcess extends CLICommand {
         } else if (/\.\d+$/.test(adapterName)) {
             this.setAdapterInstanceEnabled(adapterName, true);
         } else {
-            this.setAdapterEnabled(adapterName, true);
+            this.setAdapterEnabled(adapterName, true, args[1] === 'all');
```

Comment:
If we make it an option instead of an argument in yargs we can use `this.options.all` I think this would be cleaner and also I have in mind that this is where we want to go with cli